proto/cluster.proto

```diff
@@ -197,7 +200,16 @@ message Volume {
     HOSTTOCONTAINER = 1;
     BIDIRECTIONAL = 2;
   }
-  MountPropagationMode mount_propagation_mode = 7;
+  MountPropagationMode mount_propagation_mode = 7;  
+  // If indicate ephemeral, we need to let user specify volumeClaimTemplate
```

Comment:
Should there be a `volumeClaimTemplate` message ?

Reply:
We are specifying here `volumeClaimTemplate` parameters - storage class, storage size and access mode. The actual `volumeClaimTemplate` is build based on them

Reply:
Changed the comment to make it clear